src/Microsoft.DotNet.ProjectJsonMigration/Microsoft.DotNet.Internal.ProjectModel/ProjectReader.cs

```diff
@@ -850,11 +863,132 @@ private static void AddProjectFilesCollectionDiagnostics(JObject rawProject, Pro
                     ErrorCodes.DOTNET1015,
```

Comment:
I currently am not writing it to the ProjectMigrationReport. Will make that change.